pages/common/unzip.md

```diff
@@ -17,3 +17,7 @@
 - Extract the contents of the file(s) to stdout alongside the extracted file names:
 
 `unzip -c {{file.zip}}`
+
+- Extract zip file created by windows (you may use the command if there are non-ascii filenames in zip file):
```

Comment:
Is this option available only in windows ? Because I don't see it in `man unzip` output.

Reply:
Sorry, maybe I didn't describe it clearly. If you use windows to create a zip file which contains non-ascii character in filenames and unzip it in linux system, the file name will become unreadable. See [this](https://github.com/girder/girder/issues/985) or [this](https://github.com/rubyzip/rubyzip/wiki/Files-with-non-ascii-filenames) for more information. In this case, you should use `unzip -O gbk {{file}}`. This `-O` option is available on my `ubuntu desktop 18.04` and `ubuntu server 16.10`.

Reply:
What does the -O option do ? What does does gbk stand for ? I could not find -O here - https://linux.die.net/man/1/unzip. I hope you are not talking about -o.

Reply:
Hrm, yeah. It looks like the man page leaves something to be desired....